### PR TITLE
Tidy tempfile usage

### LINKED ISSUE #1551: Executable not clearing up after itself
We are leaving directories in `$TMPDIR` after job submission.

### PR DESCRIPTION
**Pull request by mesmith75**
Fixes #1551

### CODE REVIEW

File: ganga/GangaDirac/Lib/RTHandlers/RunTimeHandlerUtils.py [hunk 1]
```diff
@@ -86,9 +86,10 @@ def master_sandbox_prepare(app, appmasterconfig, sharedir_roots=None):
             from GangaCore.GPIDev.Lib.Job import JobError
             raise JobError("InputFiles have been requested but there are objects in the inputSandBox... Aborting Job Prepare!")
         inputsandbox = []
-        for filepattern in getInputFilesPatterns(job)[0]:
+        (fileNames, tmpDir) = getInputFilesPatterns(job)
```
Comment (alexanderrichards) [inline ganga/GangaDirac/Lib/RTHandlers/RunTimeHandlerUtils.py]: This looks like a change in the output of the `getInputFilesPatterns` i.e. from a list to a tuple(list, str). Is this change already committed as it's not part of this PR

File: ganga/GangaLHCb/Lib/RTHandlers/RTHUtils.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
 #\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\#
 import os
-import tempfile
```
Comment (alexanderrichards) [inline ganga/GangaLHCb/Lib/RTHandlers/RTHUtils.py:3]: I assume this was an unused import and removing it won't cause something to break

File: ganga/GangaLHCb/__init__.py [hunk 3]
```diff
@@ -5,7 +5,6 @@
 from os.path import exists, isdir, realpath, isfile, islink
 from os import pathsep, listdir, environ, fdopen
 import subprocess
-import tempfile
```
Comment (alexanderrichards) [inline ganga/GangaLHCb/__init__.py:8]: ditto

**Review by alexanderrichards — DISMISSED** [review]

File: ganga/GangaDirac/Lib/RTHandlers/RunTimeHandlerUtils.py (hunk 1 above)
Comment (mesmith75) [reply ganga/GangaDirac/Lib/RTHandlers/RunTimeHandlerUtils.py]: It is not a change to `getInputFilesPatterns`, more that we want to know the location of the tmpDir to remove it. The parentheses were just because I copied another bit of code that uses this function.

Comment (alexanderrichards) [reply ganga/GangaDirac/Lib/RTHandlers/RunTimeHandlerUtils.py]: sorry, my bad I missed the `[0]` on the end of the line.

**Review by alexanderrichards — APPROVED** [review]
